source/release-notes/v1.7-release-notes.rst

```diff
@@ -79,9 +105,135 @@ Passenger has been upgraded from ``5.3.7`` to ``6.0.4``. NGINX has been upgraded
+users or a subset (staff on a VPN for example).  This static page informs users
+that maintenance is underway while administrators perform disruptive tasks.
+
+See :ref:`these docs <maintenance-mode>` on how to configure and enable this feature.
+
+Titles for Files menu entries
+.............................
+
+The shortcuts in the files dropdown menu can now show an optional title along with the 
+directory path. The :ref:`original documentation <add-shortcuts-to-files-menu>` on this 
+feature has been updated to reflect the new options.
+
+
+Documentation for Keycloak as identity broker with CILogon and Keycloak with Duo 2FA
+....................................................................................
+
+Documentation for Keycloak as identity broker with CILogon has been added in the documentation
+entitled ':ref:`authentication-tutorial-oidc-keycloak-rhel7-configure-cilogon`'.
+
+Documentation for Keycloak with Duo two factor authentication has been updated in the documetnation
+entitled ':ref:`authentication-duo-2fa-with-keycloak`'.
+
+Add Grafana support for ActiveJobs app
+......................................
+
+OnDemand's ActiveJobs app can display graphs for jobs that are pulled from Grafana.
+Details on how to configure Grafana support are in :ref:`these docs <grafana-support>`.
+
+Improvements to ood-portal-generator
+....................................
+
+The ood-portal-generator app has been rewritten in Ruby and extensive testing added.
+Checksums generated for ``ood-portal.conf`` now exclude comments.
+
+Add an ssh wrapper to the shell app
+...................................
+
+An ssh wrapper script can now be used instead of just the ``ssh`` command in the
+shell app.  See how to configure this :ref:`here <ssh-wrapper>`.
+
+Regenerate ood-portal.conf with every apache restart
+....................................................
+
+Systemd will now regenerate the ood-portal.conf before every apache restart.  This means
+and administrators can now edit the ``ood_portal.yml`` configuration and then restart httpd
+with systemctl and apache will read the new configuration, because a new ood-portal.conf
+will have been written just before restarting.
+
+Support sanitizing job names
+............................
+
+Administrators can now set the ``OOD_JOB_NAME_ILLEGAL_CHARS`` environment variable to prevent
+characters from being used in job names.  For example if you do not want to use ``/`` in job
+names (as is the case with some :ref:`Grid Engine <resource-manager-sge>` versions) you would
+set ``OOD_JOB_NAME_ILLEGAL_CHARS: '/'`` in the ``pun_custom_env`` attribute of ``nginx_stage.yml``.
+
+nginx_clean support for a specific user
+.......................................
+
+The ``nginx_stage nginx_clean`` command now supports a ``-u`` or ``--user`` option so it may
+clean a specific users' PUN. For example ``nginx_stage nginx_clean -u johndoe`` would only
+clean johndoe's PUN and disregard all the others.
```

Comment:
Maybe say "so it may kill a specific user's PUN"
i.e.
kill instead of clean (even though the command is called nginx_clean which is a funny name for the kill command)